invenio_app_ils/ui/src/invenio_app_ils/common/components/ResultsTable/formatters.js

```diff
@@ -74,9 +74,7 @@ function formatEItemToTableView(eitem) {
   return {
     ID: eitem.eitem_pid ? eitem.eitem_pid : eitem.id,
     'Document ID': eitem.metadata.document_pid,
-    'Open access': eitem.metadata.open_access
-      ? eitem.metadata.open_access
-      : false,
+    'Open access': eitem.metadata.open_access ? 'Yes' : 'No',
```

Comment:
Instead of `Yes`, `No` would a boolean suffice?

Reply:
I remember that boolean values weren't displaying in the table...so imho I would say `yes`/`no` is ok :)